### PR TITLE
Add encrypt passphrase command - Closes #29

### LINKED ISSUE #29: Implement delegate passphrase encryption
In core we will be encrypting delegate passphrases and allowing decryption via API to allow forging. Lisky should include a tool to generate these passphrases and provide them to the user to use with config.json or to automatically insert them into the file.

### PR DESCRIPTION
**Pull request by willclarktech**
Closes #29

### CODE REVIEW

File: src/commands/encryptPassphrase.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		.then(handleInput)
+		.catch(handleError)
+		.then(printResult(vorpal, {}));
+};
+
+function encryptPassphraseCommand(vorpal) {
+	vorpal
+		.command('encrypt passphrase')
+		.option(...commonOptions.passphrase)
+		.option(...commonOptions.password)
```
Comment (Tosch110) [inline src/commands/encryptPassphrase.js]: Add option --json and --no-json

File: src/utils/cryptoModule.js [hunk 2]
```diff
@@ -13,7 +13,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-const lisk = require('lisk-js');
+import lisk from 'lisk-js';
```
Comment (Tosch110) [inline src/utils/cryptoModule.js]: `import { crypto } from 'lisk-js'; `
or
`import { crypto as cryptoModule } from 'lisk-js';`
could make sense in this module

Comment (willclarktech) [reply src/utils/cryptoModule.js]: Nice, I'll incorporate this in the decrypt passphrase PR as I've got a couple of refactors lined up there.

File: src/commands/encryptPassphrase.js (hunk 1 above)
Comment (willclarktech) [reply src/commands/encryptPassphrase.js]: Addressed.

**Pull request merged**